addons/binding/org.openhab.binding.gardena/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,541 @@
+                <option value="guide_3_not_found">Guide 3 not found</option>
+                <option value="difficult_finding_home">Difficult finding home</option>
+                <option value="guide_calibration_accomplished">Guide calibration accomplished</option>
+                <option value="guide_calibration_failed">Guide calibration failed</option>
+                <option value="temporary_battery_problem">Temporary battery problem</option>
+                <option value="battery_problem">Battery problem</option>
+                <option value="alarm_mower_switched_off">Alarm! Mower switched off</option>
+                <option value="alarm_mower_stopped">Alarm! Mower stopped</option>
+                <option value="alarm_mower_lifted">Alarm! Mower lifted</option>
+                <option value="alarm_mower_tilted">Alarm! Mower tilted</option>
+                <option value="connection_changed">Connection changed</option>
+                <option value="connection_not_changed">Connection not changed</option>
+                <option value="com_board_not_available">COM board not available</option>
+                <option value="slipped">Slipped</option>
+            </options>
+        </state>
+    </channel-type>
+
+    <channel-type id="sourceForNextStart" advanced="true">
+        <item-type>String</item-type>
+        <label>Source For Next Start</label>
+        <description>The source for the next start</description>
+        <state readOnly="true">
+            <options>
+                <option value="no_source">No source</option>
+                <option value="completed_cutting_daily_limit">Completed cutting daily limit</option>
+                <option value="week_timer">Week timer</option>
+                <option value="countdown_timer">Countdown timer</option>
+                <option value="mower_charging">Mower charging</option>
+                <option value="completed_cutting_autotimer">Completed cutting autotimer</option>
+                <option value="undefined">Undefined</option>
+            </options>
+        </state>
+    </channel-type>
+
+    <channel-type id="timestampNextStart">
+        <item-type>DateTime</item-type>
+        <label>Timestamp Next Start</label>
+        <description>Timestamp of the next start</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="overrideEndTime" advanced="true">
+        <item-type>DateTime</item-type>
+        <label>Override End Time</label>
+        <description>Override End Time</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="temperature">
+        <item-type>Number</item-type>
+        <label>Temperature</label>
+        <description>Temperature</description>
+        <state pattern="%d °C" readOnly="true" />
+    </channel-type>
+
+    <!-- ############################### Gateway ############################### -->
+    <thing-type id="gateway">
```

Comment:
It would be fantastic to use the gateway directly without the Gardena account, but i have not found any documentation yet. The Gardena account is also not documented, i found it with try and error. I posted my findings in the [Roboter-Forum](http://www.roboter-forum.com/showthread.php?16777-Gardena-Smart-System-Analyse)

Would be great if you can help here